gax-grpc/src/main/java/com/google/api/gax/grpc/OperationCallSettings.java

```diff
@@ -27,28 +27,25 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package com.google.api.gax.grpc;
+package com.google.api.gax.rpc;
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
 import com.google.api.core.BetaApi;
 import com.google.api.gax.retrying.TimedRetryAlgorithm;
-import com.google.longrunning.Operation;
-import com.google.protobuf.Message;
 
 /**
- * A settings class to configure an {@link OperationCallable} for calls to a long-running API method
- * (i.e. that returns the {@link Operation} type.)
+ * A settings class to configure an {@link OperationCallable} for calls to a long-running API
+ * method.
```

Comment:
Should it be "methods" (plural, since OperationCallable is usually about at least two calls: initial (API specfic) and GetOperation (gneric) for polling.